ec2/spark_ec2.py

```diff
@@ -132,7 +132,7 @@ def parse_args():
         help="Master instance type (leave empty for same as instance-type)")
     parser.add_option(
         "-r", "--region", default="us-east-1",
-        help="EC2 region zone to launch instances in")
+        help="EC2 region used to to launch instances in, or to find them in")
```

Comment:
Nit: "to to". Yes the Mima result must be a false positive given the nature of this change.